playbooks/common/openshift-cluster/additional_config.yml

```diff
@@ -1,3 +1,13 @@
+- name: Configure ntp
+  hosts:
+    - oo_nodes_to_config
+    - oo_etcd_to_config
+  vars:
+    ntp_require_clock_sync: "{{ openshift_require_clock_sync | default(false) | bool }}"
+  roles:
+    - role: ntp
+      when: not openshift.common.is_containerized | bool and openshift_manage_clock_method is defined and
+        ( openshift_manage_clock_method == "ntpd" or openshift_manage_clock_method == "auto" )
```

Comment:
I'd rename the openshfit_require_clock_sync and openshift_manage_clock_method variables to be a bit more consistent naming wise. Maybe openshift_clock_sync_required and openshift_clock_sync_method?

@abutcher thoughts on naming for these vars?